subprojects/plugins/src/main/java/org/gradle/api/distribution/Distribution.java

```diff
@@ -42,6 +43,7 @@
      * @deprecated Use #getDistributionBaseName()
      */
     @Deprecated
+    @ReplacedBy("distributionBaseName")
```

Comment:
I am not sure if this will not have an undesired side effect - as per javadoc of this annotation:
> This will cause the task <em>not</em> to be considered out-of-date when the property has changed.

This, however, makes it explicit in the DSL reference that this property is replaced with another one:
![image](https://user-images.githubusercontent.com/5560673/73159424-6c9a5000-4121-11ea-89a1-36696f1bdf21.png)